package.json

```diff
@@ -43,26 +43,29 @@
     "themes"
   ],
   "dependencies": {
+    "create-react-context": "0.2.2",
+    "create-react-ref": "0.1.0",
     "filter-react-dom-props": "0.0.2",
     "react-modal": "3.1.12"
   },
   "peerDependencies": {
-    "react": "16.3.2",
-    "react-dom": "16.3.2",
-    "classnames": "^2",
-    "lodash": "^4"
+    "react": ">=16.0.0",
```

Comment:
Since we don't have react or react-dom as a dependency, listing it here as a peerDependency should take care of matching the user's version as long as it's above v16.